### PR TITLE
Notification Popup Component

### LINKED ISSUE #26: Enable Push Notifications popup
Popup to enable push notifications.
This should probably set push notifications are enabled for the user. Not sure where to store that yet so just make the component

image

### PR DESCRIPTION
**Pull request by dafneMena**
This PR closes #26.

### Overview

This PR creates the design of the popup enable notification similar to the Figma. It adds an SVG notification bell image and includes the "Allow" and "Skip" button / text. The content is displayed within a white "container" giving it the appearance of an app. To keep consistency, the code and layout is very similar to Tim's Allow Location Access page.

Functionality of the  buttons still need to be implemented. This is solely the design as a component.

### Changes

- [x] Bell image 
- [x] Get Notification Text
- [x] Description Text
- [x] 'Allow' Button
- [x] 'Skip' Text

### Things to update

- [ ]  Functionality of the "Skip" Text
- [ ]  Functionality of the "Allow" button 

### **Preview**
Enable Notifications

### CODE REVIEW

File: client/src/components/EnableNotification/EnableNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import styles from './EnableNotification.module.scss';
+import {CustomButton } from '@/components';
+import notifBell from '../../assets/EnableNotif_bell.svg'; 
+
+interface EnableNotificationProps {
+  onAllowButtonClick: () => void;
+}
+
+// const EnableNotification: React.FC<EnableNotificationProps> = ( 
+//   onAllowButtonClick) => {
```
Comment (Linkaloo) [inline client/src/components/EnableNotification/EnableNotification.tsx:10]: remove the unnecessary comments

**Review by Linkaloo — APPROVED** [review]
Nice, looks good just make sure you don't have code commented cause its weird, ugly, and sometimes suspicious

File: client/src/components/EnableNotification/EnableNotification.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <div className={styles['EnableNotification__skipText']}>
+              <b>SKIP</b>
+            </div>
+
+            </div>
+          </div>
+        );
+};
+
+// export default EnableNotification;
```
Comment (Aligary) [inline client/src/components/EnableNotification/EnableNotification.tsx:44]: Here is another comment

**Review by Aligary — APPROVED** [review]
Great job, Dafne!

File: client/src/components/EnableNotification/EnableNotification.tsx (hunk 1 above)
Comment (dafneMena) [reply client/src/components/EnableNotification/EnableNotification.tsx]: Removed, thanks!

**Review by tbmcallister — APPROVED** [review]
I like the bell icon, and I like the well documented code. 

Good use of the CustomButton component!

this PR looks great and this component will be very useful, good job!

**Pull request merged**